Added support for Discord notifications

jakest opened the pull request:
Added support for sending notifications to Discord. Also updated the documentation with instructions.

Commit:
Added support for Discord notifications

marcone commented on setup/pi/setup-teslausb:
```diff
@@ -46,6 +46,8 @@ function read_setup_variables {
   newnamefor[ifttt_event_name]=IFTTT_EVENT_NAME
   newnamefor[ifttt_key]=IFTTT_KEY
+  newnamefor[discord_enabled]=DISCORD_ENABLED
+  newnamefor[discord_webhook_url]=DISCORD_WEBHOOK_URL
```
Don't do this.The "newnamefor" stuff is for backwards compatibility with older config files, so this isn't needed here.

marcone commented on run/send-push-message:
```diff
@@ -23,6 +23,14 @@ function send_gotify () {
+
+  curl -H "Content-Type: application/json" -d \
+    '{"username": "TeslaUSB", "content": "'"$message"'"}' \
+    $DISCORD_WEBHOOK_URL
```
shellcheck wants double quotes around the url var

jakest replied:
@marcone added

jakest committed:
Removing backwards compatibility that isn't needed

This is for backwards compatibility with older config files, so it isn't needed here.

jakest committed:
Adding double quotes to webhook URL variable

Shellcheck detected a lack of double quotes around the webhook URL var. Adding them here.

jakest commented:
@marcone the requested changes have been made. Thanks for pointing out the backwards compatibility variables.
